src/components/Menu/Menu.js

```diff
@@ -29,7 +52,40 @@ export default function Menu() {
             </ul>
           </div>
 
-          <div className="button">
+          <div className="nav-menu">
+            <img
+              src={menuIcon}
+              alt=""
+              className="hamburger"
+              onClick={onClickHandler}
```

Comment:
onClick={()=>setToggle(prevValue=>!prevValue}

Reply:
-Yes but it implements the state as className